files/scripts/supervisor-proc-exit-listener

```diff
@@ -179,6 +179,10 @@ def main(argv):
         if len(file_descriptor_list) > 0:
             line = file_descriptor_list[0].readline()
             headers = childutils.get_headers(line)
+            # Check if 'len' exists before using it
+            if 'len' not in headers:
```

Comment:
This issue was introduced by https://github.com/sonic-net/sonic-buildimage/pull/6242